src/Resources/public/js/logviewer.js

```diff
@@ -1,9 +1,16 @@
+/* DataTable */
+
 $(document).ready( function () {
     const logsTable = $('#logsTable');
     const groupsConfig = logsTable.data('groups-config');
 
     const textFormat = (value) => { return value; }
-    const jsonFormat = (value) => { return '<pre>'+JSON.parse(value)+'</pre>'; }
+    const jsonFormat = (value) => {
+        if(typeof JSON.parse(value) === 'object') {
+            return '<pre>'+JSON.stringify(value)+'</pre>';
+        }
+        return '<pre>'+JSON.parse(value)+'</pre>';
```

Comment:
Pour éviter de parser 2 fois ;) 
```suggestion
        let formattedValue = JSON.parse(value);
        if(typeof JSON.parse(value) === 'object') {
            return '<pre>'+JSON.stringify(value)+'</pre>';
        }
        return '<pre>'+formattedValue+'</pre>';
```